src/Symfony/Bundle/WebProfilerBundle/Resources/views/Profiler/layout.html.twig

```diff
@@ -73,10 +73,15 @@
                             </dd>
 
                             <dt>Profiled on</dt>
-                            <dd>{{ profile.time|date('r') }}</dd>
+                            <dd>{{ profile.time|date('r') }}
+                                <a class="btn btn-sm" href="{{ path('_profiler', { token: 'latest', id: profile.ip, method: profile.method|upper, url: profile.url }|merge(request.query.all)) }}">
+                                    {{ include('@WebProfiler/Icon/ajax.svg') }}</a>
```

Comment:
putting that in the `<dd>` of `Profiled on` looks weird to me.

Reply:
I agree with you, but I could not find a better place. At first i put it in the sidebar but then it required changes in CSS as well and was not clear what the button does.
Then i tried putting it next to the `Token` field in the header - like get the same request profile with the most recent token. Later i decided to put it where 'recent' is semi-implied, namely where the timestamp is shown. Even now it does not feel completely right but i decided to make a pull request and hopefully some day someone else with find a better place